docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,193 +1,172 @@
 
-# User                 Roles
-# -------------------- --------------
-# alice                editor, reviewer
-```
-</TabItem>
+External users are useful for larger organizations or those that want to centralize identity management via an existing SSO provider. This is often preferred for enterprise environments where identity management is handled outside Teleport.
 
-</Tabs>
+### Robots
 
-You can update the user's roles using the `tctl users update` command:
+Machine ID provides machines with an identity that can authenticate to the Teleport cluster. This identity is known as a robot. Robots are represented in Teleport by a user and a role resource and can be created via the tctl bots add command. Unlike human users, who use a password, MFA, or SSO, robot users join the cluster as Teleport services using a join method. They can still join even if local auth is disabled.
 
-<Tabs>
-<TabItem scope={"oss"} label="Teleport Community Edition">
-```code
-# Once Alice logs back in, she will be able to view audit logs
-$ tctl users update alice --set-roles=editor,auditor
-```
-</TabItem>
-<TabItem scope={["enterprise", "cloud"]} label="Commercial">
-```code
-# Once Alice logs back in, she will be able to view audit logs
-$ tctl users update alice --set-roles=editor,reviewer,auditor
-```
-</TabItem>
+See the [Machine ID introduction](../../enroll-resources/machine-id/introduction.mdx) for more information.
+
+## Teleport roles
 
-</Tabs>
+Some examples of roles include:
 
-Because Alice has two or more roles, permissions from those roles create a union. She
-will be able to act as a system administrator and auditor at the same time.
+- **access**: Grants basic access to resources.
+- **editor**: Allows users to modify configuration settings.
+- **auditor**: Allows users to view logs and auditing activities.
 
-## Step 2/3. Map SSO users to roles
+Roles enforce Role-Based Access Control (RBAC), ensuring that users only have the permissions needed for their specific tasks. 
+See [Teleport Access Controls Reference](../../reference/access-controls/roles.mdx) for in-depth information about types of roles, options, and usage.
 
-Next, follow the instructions to set up an authentication connector that maps
-users within your SSO solution to Teleport roles.
+## Understanding Teleport’s Resource Tagging and Matching
 
-### Teleport Enterprise
+Before configuring RBAC, it’s essential to understand how Teleport tags and matches resources to users’ roles.
 
-Create a SAML or OIDC application that Teleport can integrate with, then
-create an authentication connector that maps users within your application to
-Teleport roles.
+- Resource Tagging: Teleport uses labels (key-value pairs) to tag resources like SSH nodes, databases, or Kubernetes clusters. For example, a server might have the label env: production.
```

Comment:
```suggestion
- Resource Tagging: Teleport uses labels (key-value pairs) to tag resources like SSH nodes, databases, or Kubernetes clusters. For example, a server might have the label `env: production`.
```
Otherwise, this is difficult to read as regular text